pkg/onit/cli/create.go

```diff
@@ -60,6 +85,9 @@ func getCreateClusterCommand() *cobra.Command {
 			partitions, _ := cmd.Flags().GetInt("partitions")
 			partitionSize, _ := cmd.Flags().GetInt("partition-size")
 			configName, _ := cmd.Flags().GetString("config")
+			ImageTags, _ := cmd.Flags().GetStringToString("image-tags")
```

Comment:
I think it's convention to use `lowerCamel` for local variables? Or is that just in my head?

Reply:
Yes. that is correct. Will be fixed.